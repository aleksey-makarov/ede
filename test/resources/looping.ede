{
  "numbers": [1, 2, 3],
  "strings": ["foo", "bar"],
  "indexed": [false, true, false, false],
  "keyed": {
    "a": 1.0,
    "b": 2.0,
    "c": 3.0
  }
}
---
{% for n in numbers %}
  {{ n }}
{% endfor %}

{% for var in strings %}
  {{ var }}
{% endfor %}

{% for (idx, val) in indexed %}
  {{ idx }}:{{ val }}
{% endfor %}

{% for (key, val) in keyed %}
  {{ key }}:{{ val }}
{% endfor %}
