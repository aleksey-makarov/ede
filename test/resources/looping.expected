  1
  2
  3

  foo
  bar

  1:False
  2:True
  3:False
  4:False

  a:1.0
  b:2.0
  c:3.0
